appraisal.lic

```diff
@@ -10,25 +10,35 @@ class Appraisal
 
   def initialize
     @equipment_manager = EquipmentManager.new
-    @equipment_manager.wear_equipment_set?('standard')
     @settings = get_settings
+    train_list = get_settings.appraisal_training
+    
+    train_list.each do |task| 
+      break if DRSkill.getxp('Appraisal') >= 30
+      case task
+      when /zill/
```

Comment:
I wouldn't bother with a regex or multiple options per train. I think we can document the valid options on epedia.